include/constants/pokemon.h

```diff
@@ -278,7 +278,11 @@
 #define USE_RANDOM_IVS (MAX_PER_STAT_IVS + 1)
 #define MAX_PER_STAT_EVS 255
 #define MAX_TOTAL_EVS 510
+#if I_VITAMIN_EV_CAP >= GEN_8
+#define EV_ITEM_RAISE_LIMIT MAX_PER_STAT_EVS
+#elif
```

Comment:
this should be `#else`. I'm surprised it compiled

Reply:
Done. My bad.